src/card-import.js

```diff
@@ -73,111 +68,50 @@ export default class CardImport extends expose.Component {
-          },
-          DOM.input({
-            type: 'radio',
-            name: 'loadType',
-            value: 'Merge',
-            id: 'mergeChoice',
-            onChange: this.handleRadioButtonChange,
-          }),
-          DOM.label(
-            {
-              htmlFor: 'mergeChoice',
-            },
-            'Merge'
-          )
-        ),
-        DOM.div(
-          {
-            className: 'radio-button-option',
-          },
-          DOM.input({
-            type: 'radio',
-            name: 'loadType',
-            value: 'Overwrite',
-            id: 'overwriteChoice',
-            onChange: this.handleRadioButtonChange,
-          }),
-          DOM.label(
-            {
-              htmlFor: 'overwriteChoice',
-            },
-            'Overwrite'
-          )
-        )
-      ),
-      DOM.div(
-        {
-          key: 'load',
-          id: 'load',
-          className: 'button confirm-button',
-          onClick: this.handleLoadClick.bind(this),
-          style: {
-            backgroundColor: css.colors.BG,
-          },
-        },
-        'Load'
-      ),
-    ];
-  }
-
   render() {
-    return DOM.div(
-      {
-        style: {},
-      },
-      DOM.div(
-        {
-          className: 'card-title',
-        },
-        React.createElement(LeftHeaderButton, {}),
-        DOM.div(
-          {
-            className: 'card-title-text-with-arrow',
-          },
-          'Load from File'
-        ),
-        React.createElement(RightHeaderButton, {})
-      ),
-      DOM.div(
-        {
-          className: 'card-body',
-        },
-        this.renderLoadPage()
-      )
+    return (
+      <Card title="Load from File">
+        <CardSection>
+          <input
+            type="file"
+            name="fileData"
+            id="fileData"
+            className="radio-button"
+          />
+          <div className="radio-button">
+            <div className="radio-button-option">
+              <input
+                type="radio"
+                name="loadType"
+                value="Merge"
+                id="mergeChoice"
+                onChange={this.handleRadioButtonChange}
+              />
+              <label htmlFor="mergeChoice">Merge</label>
+            </div>
+            <div className="radio-button-option">
+              <input
+                type="radio"
+                name="loadType"
+                value="Overwrite"
+                id="overwriteChoice"
+                onChange={this.handleRadioButtonChange}
+              />
+              <label htmlFor="overwriteChoice">Overwrite</label>
+            </div>
+          </div>
+          <div
+            id="load"
+            className="button confirm-button"
+            onClick={this.handleLoadClick.bind(this)}
+            style={{
+              backgroundColor: css.colors.BG,
+            }}
+          >
+            Load
+          </div>
+        </CardSection>
+      </Card>
     );
```

Comment:
I only really changed this because I was using it frequently to load my test data into a memory database.